CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Backtrace Unity Release Notes
 
+## Version 3.2.6
+- `BacktraceClient` will apply sampling only to unhandled exceptions type error.
+- Fixed annotations nullable value,
+- Renamed `BacktraceUnhandledException` Classifier to `UnhandledException`
```

Comment:
- "classifier" not "Classifier"
- Let's just call it `error` and add another classifier for the log level.

Reply:
done